### ISSUE TITLE
Fix dependency on node-forge

### ISSUE THREAD

**Issue by flovouin** [GoogleCloudPlatform/iap-gcip-web-toolkit#112]
`gcip-cloud-functions` currently depends on an outdated and vulnerable version of `node-forge`, as shown by `npm audit`:

```
# npm audit report

node-forge  <=1.2.1
Severity: moderate
Improper Verification of Cryptographic Signature in `node-forge` - https://github.com/advisories/GHSA-2r2c-g63r-vccr
URL parsing in node-forge could lead to undesired behavior. - https://github.com/advisories/GHSA-gf8q-jrpm-jvxq
No fix available
node_modules/gcip-cloud-functions/node_modules/node-forge
  gcip-cloud-functions  *
  Depends on vulnerable versions of node-forge
  node_modules/gcip-cloud-functions

2 moderate severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.
```

Could you please release a version with up-to-date dependencies?

**Comment by prameshj** [comment]
Thanks! We will publish the new version soon and update this thread.

**Comment by renkelvin** [comment]
Hi @flovouin , it should be fixed in v0.1.1 now. Thanks for your report!
